bliss/tune.py

```diff
@@ -5,31 +5,29 @@
 import numpy as np
 import pytorch_lightning as pl
 import ray
+from hydra.utils import instantiate
 from omegaconf import DictConfig, OmegaConf
 from ray.tune import CLIReporter
 from ray.tune.integration.pytorch_lightning import TuneReportCallback
 from ray.tune.schedulers import ASHAScheduler
 from ray.tune.suggest import ConcurrencyLimiter
 from ray.tune.suggest.hyperopt import HyperOptSearch
 
-from bliss import sleep
-from bliss.datasets.simulated import SimulatedDataset
-
 
 def sleep_trainable(search_space, cfg: DictConfig):
     # set up the config for SleepPhase
-    cfg.model.kwargs.encoder_kwargs.channel = search_space["channel"]
-    cfg.model.kwargs.encoder_kwargs.hidden = search_space["hidden"]
-    cfg.model.kwargs.encoder_kwargs.spatial_dropout = search_space["spatial_dropout"]
-    cfg.model.kwargs.encoder_kwargs.dropout = search_space["dropout"]
+    cfg.model.encoder.channel = search_space["channel"]
+    cfg.model.encoder.hidden = search_space["hidden"]
+    cfg.model.encoder.spatial_dropout = search_space["spatial_dropout"]
+    cfg.model.encoder.dropout = search_space["dropout"]
```

Comment:
why are these lines in `tune.py` not covered by tests? Shouldn't `test_tune.py` cover them?

Reply:
Don't worry too much if it doesn't it just seemed strange to me

Reply:
I think this is related to #236. It would occasionally show as not covered due to RayTune's multiprocessing.